src/sentry/static/sentry/app/views/eventsV2/table/cellAction.tsx

```diff
@@ -113,16 +133,64 @@ export default class CellAction extends React.Component<Props, State> {
         }
         query[negation].push(`${value}`);
         break;
+      case Actions.SHOW_GREATER_THAN: {
+        // Remove query token if it already exists
+        delete query[`${column.name}`];
+        query[column.name] = [`>${value}`];
+        const field = {field: column.name, width: column.width};
+
+        // sort descending order
+        nextView = nextView.withSorts([fieldToSort(field, tableMeta, 'desc')!]);
+
+        break;
+      }
+      case Actions.SHOW_LESS_THAN: {
+        // Remove query token if it already exists
+        delete query[`${column.name}`];
+        query[column.name] = [`<${value}`];
+        const field = {field: column.name, width: column.width};
+
+        // sort ascending order
+        nextView = nextView.withSorts([fieldToSort(field, tableMeta, 'asc')!]);
+
+        break;
+      }
+      case Actions.TRANSACTION: {
+        const maybeProject = projects.find(project => {
+          return project.slug === dataRow.project;
```

Comment:
Should this be implicit return?

Reply:
done in https://github.com/getsentry/sentry/pull/19230/commits/55dd94045a8c953f5719cd9dc3907a2c174d91f6